src/sentry/constants.py

```diff
@@ -177,6 +176,14 @@ def get_all_languages():
     'mysql_pwd',
 )
 
+NOT_SCRUBBED_VALUES = set([
```

Comment:
I originally included `None` but it's actually already covered by [the first clause of sanitize](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/data_scrubber.py#L98).

I'm entirely happy to add any number of other values: almost by definition if we can think of it and add it to `NOT_SCRUBBED_VALUES` it's not sensitive! 

However:
1. I'm not that familiar with what might get to this point and what shape it would be (for example do you mean `'undefined'` the string or some exotic `undefined` object?)
2. What about `value` being a list/dict etc. Could `sanitize` be recursive?
3. I wanted to proof the concept of objects so mundane we didn't want to sanitise them even if their key was "sensitive" before going any further.